tests/examples/wibox/awidget/prompt/highlight.lua

```diff
@@ -0,0 +1,66 @@
+local parent    = ... --DOC_NO_USAGE --DOC_HIDE
+local wibox     = require( "wibox"     ) --DOC_HIDE
+local awful     = { prompt = require("awful.prompt") }--DOC_HIDE
+local beautiful = require( "beautiful" ) --DOC_HIDE
+
+    local amp = "&amp"..string.char(0x3B)
+    local quot = "&quot"..string.char(0x3B)
```

Comment:
Hasn't my tests-cleanup broke those then?!

What about having the examples in the source files, like Python doctests?
And they would be extracted and run from there?
I don't think it is bad to have the requires in there btw, since it makes it easier to adopt, doesn't it?